src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const genderMessage = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${genderMessage}`
+          : 'I am not married'}
+      </p>
```

Comment:
```suggestion
      <h2 className="Person__name">My name is {name}</h2>
      
      {age && <p className="Person__age">I am {age}</p>}
      
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${genderMessage}`
          : 'I am not married'}
      </p>
```
Use same code style like in HTML, separate adjacent elements.